cassandra/build.gradle

```diff
@@ -5,12 +5,15 @@ plugins {
 dependencies {
     api(libs.managed.datastax.cassandra.driver.core)
     api(libs.managed.datastax.cassandra.driver.mapper.processor)
+    api(libs.managed.datastax.cassandra.driver.metrics.micrometer)
```

Comment:
I don't think we should add this. Users should add it to their builds only if they want to use Micrometer integration.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-cassandra/pull/377/commits/ca0d84a6bb4152e326cbba5bd702399c7ce0db6e

Reply:
I created a starter issue to add it when generating a cassandra/micrometer project. https://github.com/micronaut-projects/micronaut-starter/issues/2139